.github/scripts/seed-protonet.sh

```diff
@@ -23,8 +23,8 @@ echo "sweet ocean blush coil mobile ten floor sample nuclear power legend where
 # fund evm-contract-deployer account (using issuance)
 kava tx issuance issue 200000000ukava kava1van3znl6597xgwwh46jgquutnqkwvwszjg04fz --from dev-wallet --gas-prices 0.5ukava -y
 
-# fund 5k kava to x/community account (using issuance)
-kava tx issuance issue 5000000000ukava kava17d2wax0zhjrrecvaszuyxdf5wcu5a0p4qlx3t5 --from dev-wallet --gas-prices 0.5ukava -y
+# fund 5k kava to x/community account
+kava tx bank send dev-wallet kava17d2wax0zhjrrecvaszuyxdf5wcu5a0p4qlx3t5 5000000ukava --gas-prices 0.5ukava -y
```

Comment:
have you confirmed you can bank send to the module account? there is also
`kava tx community fund-community-pool` which, despite its name, funds the community module account.